cursos/helm/02_helm_charts/02_practica_guiada_meiga.md

```diff
@@ -135,4 +251,3 @@ $ helm install mimeiga meiga-project/ -f misvalores.yaml
 Si tienes algún problema con esta práctica guiada puedes ver el proyecto solución completo [aquí](https://github.com/prefapp/formacion/tree/master/cursos/helm/codigo_practica_guiada_meiga/meiga-helm).
```

Comment:
@AlbertoFemenias I think you should mention the helm template option.

Reply:
Will do